pkgs/applications/editors/emacs/elisp-packages/manual-packages/codeium/default.nix

```diff
@@ -0,0 +1,35 @@
+{
+  trivialBuild,
+  fetchFromGitHub,
+  pkgs,
+  lib,
+}:
+trivialBuild {
+  pname = "codeium";
+  version = "1.6.13";
+  src = fetchFromGitHub {
+    owner = "Exafunction";
+    repo = "codeium.el";
+    rev = "1.6.13";
+    hash = "sha256-CjT21GhryO8/iM0Uzm/s/I32WqVo4M3tSlHC06iEDXA=";
+  };
+
+  buildInputs = [ pkgs.codeium ];
+
+  patches = [ ./codeium.el.patch ];
+  postPatch = ''
+    substituteInPlace codeium.el --subst-var-by codeium ${pkgs.codeium}/bin/codeium_language_server
+  '';
+
+  meta = pkgs.codeium.meta // {
+    description = "Free, ultrafast Copilot alternative for Emacs";
+    homepage = "https://github.com/Exafunction/codeium.el";
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ running-grass ];
+    platforms = pkgs.codeium.meta.platforms;
```

Comment:
This is redundant since you're already merging with codeium meta.

Reply:
I need to separate it to avoid resetting unnecessary attributes

Reply:
You're using `codeium.meta` as a base, and merging on custom things on top of that attrset: https://github.com/NixOS/nixpkgs/pull/278201/files#.

This means that `platforms` _already_ has the value of `pkgs.codeium.meta.platforms;`, and you can just remove this line altogether.

Reply:
I no longer use codeium.meta as a base. 
I have a new commit